src/calculateRentalCost.js

```diff
@@ -28,7 +28,26 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const basicDiscount = 50;
+  const liteDiscount = 20;
+  const pricePerDay = 40;
+  let totalPrice = 0;
+
+  for (let i = 0; i < days; i++) {
+    totalPrice += pricePerDay;
+  }
+
+  if (days >= 3 && days < 7) {
+    totalPrice -= liteDiscount;
+
+    return totalPrice;
+  } else if (days >= 7) {
+    totalPrice -= basicDiscount;
+
+    return totalPrice;
```

Comment:
you should avoid using else when it is not necessary. You need to check only two `if's` in this task